### ISSUE TITLE
Missing release v1.0

### ISSUE THREAD

**Issue by petebachant** [barbagroup/CFDPython#43]
For https://github.com/openjournals/jose-reviews/issues/21.

**Comment by gforsyth** [comment]
I think it makes sense to hold off on cutting a release until we resolve the other issues from review -- that way the tag position can reflect the paper most accurately.

**Comment by petebachant** [comment]
Makes sense. Just wanted to throw up an issue since it's part of the review.

**Comment by petebachant** [comment]
Notebooks 0 and 1 have version numbers in them as well. These should probably be automatically inserted using `git describe`, if they're even necessary at all.
